ballerina/tests/hub_client_test.bal

```diff
@@ -40,13 +41,36 @@ service /callback on new http:Listener(9094) {
         return caller->respond("Content Delivery Success");
     }
 
+    isolated resource function post successUrlEncoded(http:Caller caller, http:Request req) returns error? {
+        io:println("Hub Content Distribution message received : ", req.getTextPayload());
+        http:Response res = new;
+        res.setTextPayload("status=success", mime:APPLICATION_FORM_URLENCODED);
+        return caller->respond(res);
```

Comment:
Update theses resources to the newer syntax. May be as a separate task.

Reply:
Will capture this in following issue [1] 

[1] - [https://github.com/ballerina-platform/ballerina-standard-library/issues/2539](https://github.com/ballerina-platform/ballerina-standard-library/issues/2539)